start-kind.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+CONTAINERS="tigrisdata/tigris:1.0.0-beta.18
```

Comment:
Do you expect this variable to be modified in the future, from say, CI? Might be beneficial to split these into individual containers to control component versions.

Reply:
I can do that in the future. Right now the reason for it is that otherwise all kind nodes download all containers, so with the default limits, you can only start this twice a day. So for now the main goal is to spare some bandwidth and docker pulls. We can do versioning of individual components later.